grammar.js

```diff
@@ -262,6 +265,28 @@ module.exports = grammar({
     keyword_cost: _ => make_keyword("cost"),
     keyword_rows: _ => make_keyword("rows"),
     keyword_support: _ => make_keyword("support"),
+    keyword_extension: _ => make_keyword("extension"),
+    keyword_out: _ => make_keyword("out"),
+    keyword_inout: _ => make_keyword("inout"),
+    keyword_variadic: _ => make_keyword("variadic"),
+
+    keyword_session: _ => make_keyword("session"),
+    keyword_isolation: _ => make_keyword("isolation"),
+    keyword_level: _ => make_keyword("level"),
+    keyword_serializable: _ => make_keyword("seria"),
```

Comment:
is this called seria? I have search pg and mariadb docs and was not able to find this. Can you give me an example?

Reply:
It is a typo.